package.json

```diff
@@ -26,13 +26,13 @@
     "color": "^3.0.0",
     "date-fns": "^1.29.0",
     "email-validator": "^2.0.4",
-    "fp-ts": "^1.12.2",
+    "fp-ts": "1.12.0",
     "hoist-non-react-statics": "^3.0.1",
     "instabug-reactnative": "8.2.2",
-    "io-ts": "^1.8.5",
+    "io-ts": "1.8.5",
     "io-ts-types": "^0.4.7",
-    "italia-pagopa-commons": "^0.0.2",
-    "italia-ts-commons": "^5.1.0",
+    "italia-pagopa-commons": "^1.0.0",
```

Comment:
I've upgraded `italia-pagopa-commons` to depend of the same version of `io-ts` that we use here